### PR TITLE
Show data cap after receiving first bandwidth update

### PR DESCRIPTION
**Pull request by atavism**
The data cap should show up right away now for free users (if it's enabled) with these changes

### CODE REVIEW

File: lib/features/vpn/vpn_bandwidth.dart [hunk 1]
```diff
@@ -7,9 +7,13 @@ class VPNBandwidth extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return sessionModel
-        .bandwidth((BuildContext context, Bandwidth bandwidth, Widget? child) {
-          // User does not have bandwidth cap off
-      if (bandwidth.remaining > 0 ) {
+        .bandwidth((BuildContext context, Bandwidth? bandwidth, Widget? child) {
+      return sessionModel.proUser((context, isProUser, child) {
+        if (bandwidth == null || isProUser) {
```
Comment (atavism) [inline lib/features/vpn/vpn_bandwidth.dart:12]: This is the key change. I added an extra check here as well just to make sure we never show the data cap to Pro users

**Comment by jigar-f** [comment]
Thanks, @atavism for making these changes, I'll test changes tomorrow.

**Review by jigar-f — APPROVED** [review]
LGTM

**Comment by atavism** [comment]
Thanks for the review, @jigar-f!

**Pull request merged**